cmd/idracsdkapps/elasticsearch/elasticsearchplugin.go

```diff
@@ -0,0 +1,336 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/elastic/go-elasticsearch/v8/esapi"
+	"github.com/elastic/go-elasticsearch/v8"
+)
+
+var mtx sync.Mutex
+var dat map[string]interface{}
+var reportCount = 0
+var invalidJsonPkts = 0
+
+var telemetryserverIP = "100.98.72.157"

```

Comment:
ip should not be hard coded in.

Reply:
May be i just remove up the iDRAC IP and credentials and then push for now. And all others later.